Automattic/jetpack
# tools/version-packages.sh: Check for missed upgrades

#### Changes proposed in this Pull Request:

If a package isn't upgraded in a run of tools/version-packages.sh, then
something is wrong (e.g. p9dueE-4bx-p2). So check for that and error out
if it happens.

#### Jetpack product discussion


p1641996471074300-slack-C01U2KGS2PQ

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Edit `projects/plugins/jetpack/composer.json` to do minor or major bumps of some package versions, then run `tools/version-packages.sh projects/plugins/jetpack/`. It should print relevant errors and exit with a non-zero status.
* If you don't bump any versions, it should complete successfully.
